### PR TITLE
circlecross redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
god, the og code was absolutely awful

**Comment by SansPapyrus683** [comment]
wait remind me to redo the editorial for sol 3 as well

File: solutions/bronze/usaco-712.mdx [hunk 1]
```diff
@@ -35,225 +27,422 @@ using namespace std;
...
+			}
+		}
 	}
-	int ans = 0;
-	for (int i = 0; i < 26; ++i)
-		for (int j = 0; j < 26; ++j)
-			ans += Start[i] < Start[j] && Start[j] < End[i] && End[i] < End[j];
-	cout << ans << endl;
+	
+	freopen("circlecross.out","w",stdout);
```
Comment (ChuangSheep) [inline solutions/bronze/usaco-712.mdx:53]:
```suggestion
	freopen("circlecross.out", "w", stdout);
```

**Review by ChuangSheep — APPROVED** [review]
nice 🎉

**Review by ryanchou-dev — APPROVED** [review]
yessir

**Pull request merged**